test/com/facebook/buck/apple/project_generator/ProjectGeneratorTest.java

```diff
@@ -4493,6 +4493,45 @@ public void testAppBundleDoesntLinkFrameworkWrappedWithResource() throws Excepti
     assertThat(target.getBuildPhases().size(), Matchers.equalTo(1));
   }
 
+  @Test
+  public void testGeneratedProjectStructureAndSettingsWithBridgingHeader() throws IOException {
+    BuildTarget buildTarget = BuildTarget.builder(rootPath, "//foo", "lib").build();
+    TargetNode<?> node = AppleLibraryBuilder
+        .createBuilder(buildTarget)
+        .setConfigs(
+            Optional.of(
+                ImmutableSortedMap.of(
+                    "Debug",
+                    ImmutableMap.<String, String>of())))
+        .setSrcs(Optional.of(ImmutableSortedSet.<SourceWithFlags>of()))
+        .setBridgingHeader(Optional.<SourcePath>of(new FakeSourcePath("BridgingHeader/header1.h")))
+        .build();
+
+    ProjectGenerator projectGenerator = createProjectGeneratorForCombinedProject(
+        ImmutableSet.<TargetNode<?>>of(node));
+
+    projectGenerator.createXcodeProjects();
+
+    // check if bridging header file existing in the project structure
+    PBXProject project = projectGenerator.getGeneratedProject();
+    PBXGroup targetGroup =
+        project.getMainGroup().getOrCreateChildGroupByName(buildTarget.getFullyQualifiedName());
+    PBXGroup sourcesGroup = targetGroup.getOrCreateChildGroupByName("Sources");
+
+    assertThat(sourcesGroup.getChildren(), hasSize(1));
+
+    PBXFileReference fileRefBridgingHeader =
+        (PBXFileReference) Iterables.get(sourcesGroup.getChildren(), 0);
+    assertEquals("header1.h", fileRefBridgingHeader.getName());
+
+    // check for bridging header build setting
+    PBXTarget target =
+        assertTargetExistsAndReturnTarget(project, "//foo:lib");
+    ImmutableMap<String, String> buildSettings = getBuildSettings(buildTarget, target, "Debug");
+    assertEquals("$(SRCROOT)/../BridgingHeader/header1.h",
```

Comment:
`..` is there because the output folder is fixed: https://github.com/nguyentruongtho/buck/blob/eba8551d87158e76ee395ddc9fd88cea7dc9fbd1/test/com/facebook/buck/apple/project_generator/ProjectGeneratorTest.java#L157-L157, a bit weird, but its fine.